CSS/style.css

```diff
@@ -114,3 +114,78 @@
 
 }
 
+.second-section {
+    max-width: 1440px;
+    min-height: 672px;
+    background-color: #FFFFFF;
+    
```

Comment:
- [ ] Any extra lines of code is reduces the readability and performance of your code so if `background-color` is already set white don't need to apply it in small sections. 
- [ ] It's not a best way to specify `min-height` property here instead you add `padding-bottom` or `margin-bottom` for example, to add spaces.